tests/Doctrine/Tests/DBAL/Functional/Driver/PDOPgsqlConnectionTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * PDOPgsqlConnectionTest.php
+ * Created by William Schaller
+ * Date: 4/2/2015
+ * Time: 12:08 PM
+ */
+namespace Doctrine\Tests\DBAL\Functional\Driver;
+
+use Doctrine\DBAL\Platforms\PostgreSqlPlatform;
+use Doctrine\Tests\DbalFunctionalTestCase;
+use Doctrine\Tests\TestUtil;
+
+class PDOPgsqlConnectionTest extends DbalFunctionalTestCase
+{
+    protected function setUp()
+    {
+        if ( ! extension_loaded('pdo_pgsql')) {
+            $this->markTestSkipped('pdo_pgsql is not loaded.');
+        }
+
+        parent::setUp();
+
+        if ( ! $this->_conn->getDatabasePlatform() instanceof PostgreSqlPlatform) {
+            $this->markTestSkipped('PDOPgsql only test.');
+        }
+    }
+
+    public function testPdoPgsqlConnectionWithUTF8Charset()
+    {
+        $GLOBALS['db_charset'] = 'UTF8';
```

Comment:
I don't think it is a good idea to change global state here as all following tests will be affected by this.